lib/generators/rails/decanter_generator.rb

```diff
@@ -2,12 +2,12 @@ module Rails
   module Generators
     class DecanterGenerator < NamedBase
       source_root File.expand_path('../templates', __FILE__)
-      check_class_collision :suffix => 'Decanter'
+      check_class_collision suffix: 'Decanter'
       ASSOCIATION_TYPES = [:has_many, :has_one, :belongs_to]
 
-      argument :attributes, :type => :array, :default => [], :banner => 'field:type field:type'
+      argument :attributes, type: :array, default: [], banner: 'field:type field:type'
 
-      class_option :parent, :type => :string, :desc => 'The parent class for the generated decanter'
+      class_option :parent, type: :string, desc: 'The parent class for the generated decanter'
```

Comment:
Are these changes related to this new functionality?

Reply:
nope - was just flipping through files to get my bearing and it stuck out to me